SHIR/build.ps1

```diff
@@ -40,7 +40,7 @@ function Install-Jre() {
     Invoke-WebRequest "https://api.adoptium.net/v3/installer/latest/11/ga/windows/x64/jdk/hotspot/normal/eclipse?project=jdk" -OutFile "C:\SHIR\OpenJdk11.msi"
     Start-Process -Wait -FilePath msiexec -ArgumentList /i, "C:\SHIR\OpenJdk11.msi", "ADDLOCAL=FeatureMain,FeatureEnvironment,FeatureJarFileRunWith,FeatureJavaHome", 'INSTALLDIR="C:\Program Files\Java"', /quiet -Verb RunAs
     Write-Log "OpenJDK 11 installed successfully"
-    [Environment]::SetEnvironmentVariable("Path", $env:Path + ";C:\Program Files\Java", "Machine")
+    [Environment]::SetEnvironmentVariable("Path", $env:Path + ";C:\Program Files\Java;C:\Program Files\Java\bin", "Machine")
```

Comment:
Adding both the root JAVA_HOME and the /bin because it was working before with a previous version, so this will have greater compatibility.